scripts/cspell/spellcheck.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if ! sort -c project-words.txt; then \
+    echo "project-words.txt is not sorted." && exit 1; \
+fi
+npm install cspell
+npm run spellcheck
```

Comment:
in which directory will it run?

Reply:
It is being called from the `scripts/cspell/` directory. But the spellcheck script in `package.json` traverses to the content folder and performs a spellcheck on all the markdown files inside it.